crates/sargon/src/signing/tx_to_sign.rs

```diff
@@ -1,20 +1,28 @@
 use crate::prelude::*;
 
-#[derive(Clone, Debug, PartialEq, Eq, std::hash::Hash)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub(crate) struct TXToSign {
-    pub(crate) intent_hash: IntentHash,
+    pub(crate) intent: TransactionIntent,
     entities_requiring_auth: Vec<AccountOrPersona>, // should be a set but Sets are not `Hash`.
```

Comment:
ah, you can now change `Vec<AccountOrPersona>` into an `IndexSet<AccountOrPersona>` :)